firmware/application/apps/ais_app.hpp

```diff
@@ -127,115 +119,94 @@ class AISRecentEntryDetailView : public View {
 	void focus() override;
 	void paint(Painter&) override;
 
-    AISRecentEntryDetailView(const AISRecentEntryDetailView&Entry);
-    AISRecentEntryDetailView &operator=(const AISRecentEntryDetailView&Entry);
+	AISRecentEntryDetailView(const AISRecentEntryDetailView& Entry);
+	AISRecentEntryDetailView& operator=(const AISRecentEntryDetailView& Entry);
 
-private:
-	AISRecentEntry entry_ { };
+ private:
+	AISRecentEntry entry_{};
 
-	Button button_done {
-		{ 125, 224, 96, 24 },
-		"Done"
-	};
-	Button button_see_map {
-		{ 19, 224, 96, 24 },
-		"See on map"
-	};
-	GeoMapView* geomap_view { nullptr };
-	bool send_updates { false };
-
-	Rect draw_field(
-		Painter& painter,
-		const Rect& draw_rect,
-		const Style& style,
-		const std::string& label,
-		const std::string& value
-	);
+	Button button_done{{125, 224, 96, 24}, "Done"};
+	Button button_see_map{{19, 224, 96, 24}, "See on map"};
+	GeoMapView* geomap_view{nullptr};
+	bool send_updates{false};
+
+	Rect draw_field(Painter& painter,
```

Comment:
Another really strange case :(

Reply:
Line length

Reply:
<img width="635" alt="image" src="https://github.com/eried/portapack-mayhem/assets/21130052/92287354-73c4-4c0a-a85c-4ca30ff9f262">

I'll have to pull this and check with a real diff tool when I get home. This looks broken on GitHub.

Reply:
I sent you a message on discord with something that will help